Utilities/VisualStudioProjectTemplates/EdFi.ProjectTemplates.Extensions/EdFi.Ods.Extensions.ExtensionName.nuspec

```
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>EdFi.Ods.Extensions.ExtensionName</id>
+    <version>5.1.0</version>
+    <authors>Ed-Fi Alliance</authors>
+    <owners>Ed-Fi Alliance</owners>
+    <projectUrl>https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>EdFi.Ods.Extensions.ExtensionName</description>
+    <releaseNotes></releaseNotes>
+  </metadata>
+  <files>
+    <file src="assemblyMetadata.json" target="."/>
+    <file src="bin\$configuration$\netcoreapp3.1\EdFi.Ods.Extensions.ExtensionName.dll" target="."/>
+  </files>
```

I see we're using `$safeprojectname$` which gets replaced with the project name once you have created it. Once you have this file setup to be created with an extension project using the template can you test and see if something like below would work? It would be especially nice to have for the dll file name.
```suggestion
  <metadata>
    <id>$safeprojectname$</id>
    <version>1.0.0</version>
    <authors>Ed-Fi Alliance</authors>
    <owners>Ed-Fi Alliance</owners>
    <projectUrl>https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation</projectUrl>
    <description>$safeprojectname$</description>
  </metadata>
  <files>
    <file src="assemblyMetadata.json" target="."/>
    <file src="bin\$configuration$\netcoreapp3.1\$safeprojectname$.dll" target="."/>
  </files>
```

---

I also noticed that we have created the `Artifacts/` folder structure in the template project but this is not included in the output. For this nuspec file specifically it would be nice to have that directory included by default similar to our extensions:
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Extensions/blob/46c684943b728384f6aa3959136e46eff2afbc0f/Extensions/EdFi.Ods.Extensions.Sample/EdFi.Ods.Extensions.Sample.nuspec#L16

---

> I see we're using `$safeprojectname$` which gets replaced with the project name once you have created it. Once you have this file setup to be created with an extension project using the template can you test and see if something like below would work? It would be especially nice to have for the dll file name.

I update it now . Thanks

---

> I also noticed that we have created the `Artifacts/` folder structure in the template project but this is not included in the output. For this nuspec file specifically it would be nice to have that directory included by default similar to our extensions:
> https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Extensions/blob/46c684943b728384f6aa3959136e46eff2afbc0f/Extensions/EdFi.Ods.Extensions.Sample/EdFi.Ods.Extensions.Sample.nuspec#L16

I inculded Artifacts/` folder in nuspec file now